automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookCustomizationSpec.scala

```diff
@@ -100,7 +101,7 @@ final class NotebookCustomizationSpec extends FreeSpec
           withWebDriver { implicit driver =>
             withNewNotebook(cluster) { notebookPage =>
               val query = """!jupyter labextension install @jupyterlab/toc"""
-              val result = notebookPage.executeCell(query).get
+              val result = notebookPage.executeCell(query, timeout = 5.minutes).get
```

Comment:
Bumped because we saw some timeouts in tests on the `!jupyter labextension install @jupyterlab/toc` cell.

Reply:
👍